### DISCUSSION TITLE
How to specify postgres connection pool size?

### DISCUSSION THREAD

**Discussion by ppatrzyk** [plausible/analytics#1591] [Self-Hosted Support] [answered]
Hello,

I'd like to decrease the number of connections Plausible makes to Postgres database. How to achieve that? I'm running Plausible v1.4.3 as a docker container. 

I don't see anything in your [docs](https://plausible.io/docs/self-hosting-configuration). I found some older [issue](https://github.com/plausible/analytics/issues/198) and [PR](https://github.com/plausible/analytics/pull/89/files) - I tried to set `DATABASE_POOLSIZE` env variable but it didn't change anything.

Thank you for help

**Comment by ukutaht** [comment, marked as answer]
It can be added to the DATABASE_URL variable as a query parameter. Try this

```
DATABASE_URL=postgres://postgres:postgres@plausible_db:5432/plausible_db?pool_size=5
```

**Reply by ppatrzyk** [reply]
works, thanks!
